WorkloadManagementSystem/Utilities/JobMemory.py

```diff
@@ -13,7 +13,8 @@ def getJobFeatures():
                 'jobstart_secs', 'mem_limit_MB', 'allocated_CPU ', 'shutdowntime_job' ):
     fname = os.path.join( os.environ['JOBFEATURES'], item )
     try:
-      val = urllib.urlopen( fname ).read()
+      with open( fname, "r" ) as f:
```

Comment:
I don't think it is a local file necessarily... For VMs the way Andrew recommends to set it up is with http access, therefore this is why everywhere we use urllib.